src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js

```diff
@@ -71,12 +71,7 @@
             const invalidLatitude = allEmptyOrFilledResult.invalidInputType === POSITION_TYPE_LATITUDE;
 
             if (latResult.isError || (!isNativeEvent && invalidLatitude)) {
-                if (invalidLatitude && !latResult.isError) {
-                    latResult.isError = true;
-                    latResult.errorMessage = allEmptyOrFilledResult.errorMessage;
-                }
-
-                latInput.dispatchEvent(new Event('showLatitudeError'));
+                return false;
```

Comment:
This should return the object with `isError = false`

Reply:
Nope